src/components/Main/Header/Header.tsx

```diff
@@ -9,11 +10,44 @@ interface Props {
 
 function Header(props: Props) {
   return (
-    <div id={props.id}>
-      <h2>HELLO WORLD</h2>
-      <h1>WE'RE MAKING WAVES</h1>
-      <h2>GWC at UVA</h2>
-      <img src="../../assets/images/main-header-wave.svg" alt="Orange wave" />
+    <div id={props.id}> 
+
+    <div className="container-fluid m-0 bg-orange">
+      <Row lg={2}>
```

Comment:
instead of using row/col for spacing, use class names https://getbootstrap.com/docs/4.1/utilities/spacing/

Reply:
Used class names for the horizontal spacing, but the vertical spacing wasn't enough with the classnames (since I believe they only go up to 5). I'm still trying to figure out why the background html background is a little weird in the mobile version.
![image](https://user-images.githubusercontent.com/59893188/116476529-58c02b00-a849-11eb-91c8-de86bb673ffa.png)
![image](https://user-images.githubusercontent.com/59893188/116476587-6d9cbe80-a849-11eb-8005-cccd7e1dba74.png)